2つのセルを表示し、処理できるようにする

glact1973 opened the pull request.

glact1973 committed:
2つのセルを表示し、処理できるようにする

num025 commented:
eslintは導入されていますか？
コマンドラインで以下のコマンドを打つことでインストールできます。
```
$ npm install -g eslint
```

eslintを使用することで、セミコロンの打ち忘れやインデントの崩れなどをわかりやすく指摘してくれます。
このリポジトリにはすでにルールセットが存在するので、まずはeslintに指摘されたコーディングのミスを直してみていただければと思います。

参考リンク:
https://eslint.org/
https://qiita.com/mysticatea/items/f523dab04a25f617c87d

glact1973 committed:
2つのセルを表示し、処理できるようにする

Eslintで体裁を修正（Eslintの存在自体、初めて知りました・・・）

なお、event.target.idのeventに波線がついています。eventは利用しない方が良いのでしょうか？
また、その場合、2つ正解がある今回の場合、クリックする順番が順不同のため、event.target.id以外にクリックしたセルの位置を認識する方法が見つかりません。（色々トライしたのですが見つからず・・・）
（場所の認識をしないと、最初に赤くしたセルをクリックすると、後に赤くなったセルが消える）
別にクリックした要素を把握する方法を教えていただきたく・・・

glact1973 commented:
Eslintで体裁を修正（Eslintの存在自体、初めて知りました・・・）

なお、event.target.idのeventに波線がついています。eventは利用しない方が良いのでしょうか？
また、その場合、2つ正解がある今回の場合、クリックする順番が順不同のため、event.target.id以外にクリックしたセルの位置を認識する方法が見つかりません。（色々トライしたのですが見つからず・・・）
（場所の認識をしないと、最初に赤くしたセルをクリックすると、後に赤くなったセルが消える）
別にクリックした要素を把握する方法を教えていただきたく・・・

Mitsuru53 commented:
eventが宣言されていないせいですね！

```
handleActiveCellClick(event) { // ここにeventを入れてみてください
  // クリックしたセルのエレメント（ID）を取得し、次処理の引数として渡す
  const activeCellPosition = event.target.id;
  // this.callbacks.handleActiveCellSelected();
  this.callbacks.handleActiveCellSelected(activeCellPosition);
}
```

glact1973 committed:
最終アップ

glact1973 commented:
なるほど、確かにそうですね。再アップしました。
